src/js/components/Drop/DropContainer.js

```diff
@@ -123,30 +123,24 @@ export class DropContainer extends Component {
     const { align, dropTarget, responsive, stretch, theme } = this.props;
     const windowWidth = window.innerWidth;
     const windowHeight = window.innerHeight;
-
     /* eslint-disable-next-line react/no-find-dom-node */
     const target = findDOMNode(dropTarget);
     /* eslint-disable-next-line react/no-find-dom-node */
     const container = findDOMNode(this.dropRef.current);
     if (container && target) {
       // clear prior styling
-      container.style.left = '';
```

Comment:
Why are we not clearing the inline styling that is set at the bottom of this function? The reason to clear anything set earlier is to allow any resize or repopulation of content to reset the calculations without being influenced by prior settings.

Reply:
See the original issue associated. By clearing top the browser will scroll to the top automatically

Reply:
But, if the contents of the drop changes, we will no longer reposition it correctly as `place()` will be called with the prior values, instead of taking the new values resulting from the new content in the drop.

Reply:
Have you tested it? I tried and the resize seems to be working. I will look into it again

Reply:
It's not just the resize, it's also changing contents of the drop, like when the back-end is slow to retrieve content, coupled with needing to tweak the drop placement due to the size of the contents changing and the location requiring adjustment due to colliding with the window boundaries.

Reply:
I try it again. Do you have other ideas on how to solve the scroll to top problem?

Reply:
It seems like the issue is that the contents can change for a variety of reasons. For example, adding or removing content when lazy loading, or based on inter-relationships between options when some are selected vs. just rendering the selected ones differently. Maybe we should explore calculating the desired size before rendering and then after rendering re-calculating it and seeing if it changed. If it changed, we place(), if it didn't change, we don't place()?

Reply:
I explored this solution initially, but quickly gave up on it since you have have scrolled down clicked something that affected the size and you still want to stay in the same scroll position. I think calling place should not cause the browser to scroll to the top. I’m trying to explore solutions that will allow that to happen

Reply:
fixed